core/src/main/res/drawable/ic_disc_full.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:width="20dp"
+    android:height="16dp"
+    android:viewportWidth="20"
+    android:viewportHeight="16">
+    <path
+        android:pathData="M17.1 11.5s0-1.7 0-1.7s1.8 0 1.8 0s0 1.7 0 1.7s-1.8 0-1.8 0zm0-8s1.8 0 1.8 0s0 4.5 0 4.5s-1.8 0-1.8 0s0-4.5 0-4.5zm-8.9-2.6c2 0 3.6 0.7 5 2.1s2.1 3 2.1 5s-0.7 3.7-2.1 5s-3 2.1-5 2.1c-1.9 0-3.6-0.7-5-2.1s-2.1-3-2.1-5s0.7-3.6 2.1-5s3.1-2.1 5-2.1zm0 8.9c0.5 0 0.9-0.2 1.3-0.5s0.5-0.8 0.5-1.3s-0.2-0.9-0.5-1.3s-0.8-0.5-1.3-0.5s-0.9 0.2-1.2 0.5s-0.6 0.8-0.6 1.3s0.2 0.9 0.6 1.3s0.7 0.5 1.2 0.5z"
+        android:fillColor="#454545" />
```

Comment:
When setting a static color, it will not work in dark mode

```suggestion
        android:fillColor="?android:attr/textColorPrimary" />
```

Reply:
ah. good to know. Would you recommend the same for [md_info_outline.xml](https://github.com/AntennaPod/AntennaPod/pull/6578/files#diff-4fd949172282045e330eafe33e19280920c08ac9dd5fefdca578a7758eccda84)

Reply:
I think there might already be an info icon in the project somewhere. Not sure if it is just the one on the podcast screen (which is always white, so does not fit the use here) or if there is an additional one that follows the font color

Reply:
Assuming that to be the case. Will push with both changed. That makes the disc full slightly darker than it was in the original, but I'm going to assume that's okay.

![image](https://github.com/AntennaPod/AntennaPod/assets/1500358/d4da489c-94b7-4a88-950a-5c4fb3872b62)

Reply:
> I think there might already be an info icon in the project somewhere. Not sure if it is just the one on the podcast screen (which is always white, so does not fit the use here) or if there is an additional one that follows the font color

Interesting point. In fact there are two. `ic_info.xml` and `ic_info_white.xml`. Assuming you were interested in me trying the former. Here's a screenshot with how it looks. You'll note that it is larger than the one I matched to the current info icon.

![image](https://github.com/AntennaPod/AntennaPod/assets/1500358/35e1b439-8325-43a1-9735-79fa51bed9e9)

Reply:
Then I would prefer using the existing icon. Changing the size seems cumbersome using CompoundDrawables, so you could also add an ImageView to the layout that shows the "filtered" message (if I remember correctly, the file should be called something like `nav_divider.xml`). That would then also avoid having to manually add a space to the string - then you can use a margin on the TextView.

Reply:
I've done that change as best I can (I'm learning as I go here). If a LinearLayout wasn't the best way to do it, please shout.

I had to do a bit of extra testing because now we need to make sure that the icon is hidden as well as the text. Below are two images. 

The first is text and icon being hidden (nothing to see) 

![hidden-filtered-messages](https://github.com/AntennaPod/AntennaPod/assets/1500358/96bd1fd6-b8db-4b16-81ef-9534ecfda274)

the second is the new layout with the icon roughly the right size, location and offset from the text to match current behaviour (see images earlier in the commentary for comparison).

![filtered-sub-icon-as-image-view](https://github.com/AntennaPod/AntennaPod/assets/1500358/6363911e-d7bd-4bfb-93fe-162115f3cba8)